lib/byron/test/integration/Cardano/Wallet/Byron/Faucet.hs

```diff
@@ -11,11 +11,12 @@ import Prelude
 import Control.Concurrent.MVar
     ( newMVar )
 import Test.Integration.Faucet
-    ( Faucet (..), icaMnemonics, rndMnemonics, seqMnemonics )
+    ( Faucet (..), icaMnemonics, mirMnemonics, rndMnemonics, seqMnemonics )
 
 initFaucet :: IO Faucet
 initFaucet = Faucet
     <$> newMVar seqMnemonics
     <*> newMVar icaMnemonics
     <*> newMVar rndMnemonics
+    <*> newMVar mirMnemonics
```

Comment:
:+1:

Reply:
I generated 10 of them, used 2 or 3 I think in the scenarios I wrote, so there is still some room for some :p